graylog2-server/src/main/java/org/graylog2/alerts/StaticEmailAlertSender.java

```diff
@@ -204,9 +216,22 @@ public void sendEmails(Stream stream, AlertCondition.CheckResult checkResult, Li
         if(stream.getAlertReceivers().get("emails") != null) {
             for (String email : stream.getAlertReceivers().get("emails")) {
                 if(!email.isEmpty()) {
-                    sendEmail(email, stream, checkResult, backlog);
+                    recipients.add(email);
                 }
             }
         }
+
+        if (recipients.size() == 0) {
+            final Notification notification = notificationService.buildNow()
+                    .addNode(nodeId.toString())
+                    .addType(Notification.Type.GENERIC)
+                    .addSeverity(Notification.Severity.NORMAL)
+                    .addDetail("title", "Stream \"" + stream.getTitle() + "\" is alerted, but no recipients have been defined!")
```

Comment:
I would change the exclamation mark to a full-stop. Sounds less aggressive/panicky.